vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -186,11 +364,161 @@ EXPORT(int, sceGxmAddRazorGpuCaptureBuffer) {
     return UNIMPLEMENTED();
 }
 
+EXPORT(void, sceGxmSetDefaultRegionClipAndViewport, SceGxmContext *context, uint32_t xMax, uint32_t yMax) {
+    const std::uint32_t xMin = 0;
+    const std::uint32_t yMin = 0;
+
+    context->state.viewport.offset.x = 0.5f * static_cast<float>(1.0f + xMin + xMax);
+    context->state.viewport.offset.y = 0.5f * (static_cast<float>(1.0 + yMin + yMax));
+    context->state.viewport.offset.z = 0.5f;
+    context->state.viewport.scale.x = 0.5f * static_cast<float>(1.0f + xMax - xMin);
+    context->state.viewport.scale.y = -0.5f * static_cast<float>(1.0f + yMax - yMin);
+    context->state.viewport.scale.z = 0.5f;
+
+    context->state.region_clip_min.x = xMin;
+    context->state.region_clip_max.x = xMax;
+    context->state.region_clip_min.y = yMin;
+    context->state.region_clip_max.y = yMax;
+    context->state.region_clip_mode = SCE_GXM_REGION_CLIP_OUTSIDE;
+
+    if (context->alloc_space) {
```

Comment:
When does this thing become null?

Reply:
Just a safe check

Reply:
It may be null sometimes, when deferred context has not start recording. State can still be set before it.

Reply:
ah deferred init it later.